app/models/comment.rb

```diff
@@ -0,0 +1,8 @@
+class Comment < ApplicationRecord
+  belongs_to :post
+  belongs_to :user
+
+  def update_comment_counter
+    post.update(comments_counter: post.comments.count)
+  end
+end
```

Comment:
[OPTIONAL] It is a good practice to run methods like this with rails callbacks like `:after_save` and make the method private.